docs/user_guide/hopsworks/featurestore.rst

```diff
@@ -504,17 +504,42 @@ In the AWS management console go to *IAM*, select *Roles* and then the role that
+**(Alternative 2) Storing the API Key in the AWS Stystems Manager Parameter Store**
 
-    !pip install hops
+In the AWS management console ensure that you active region is the region you use for SageMaker. Go to the *AWS Systems Manager* choose *Parameter Store* and select *Create Parameter*. As name enter */hopsworks/project/[MY_HOPSWORKS_PROJECT]/role/[MY_SAGEMAKER_ROLE]/type/api-key* replacing [MY_HOPSWORKS_PROJECT] with the name of the project hosting the Feature Store in Hopsworks and [MY_SAGEMAKER_ROLE] with the AWS role used by the SageMaker instance that should access the Feature Store. Select *Secure String* as type and create the parameter.
+
+.. _hopsworks_parameter_store.png: ../../_images/parameter_store.png
+.. figure:: ../../imgs/feature_store/parameter_store.png
+    :alt: Hopsworks feature store parameter store
+    :target: `hopsworks_parameter_store.png`_
+    :align: center
+    :scale: 20 %
+    :figclass: align-center
+
+**(Alternative 2) Granting access to the secret to the SageMaker notebook role**
+
+In the AWS management console go to *IAM*, select *Roles* and then the role that is used when creating SageMaker notebook instances. Select *Add inline policy*. Choose *Systems Manager* as service, expand the *Read* access level and check *GetParameter*. Expand Resources and select *Add ARN*. Fill in the region of the *Systems Manager* as well as the name of the parameter **WITHOUT the leading slash** e.g. *hopsworks/project/[MY_HOPSWORKS_PROJECT]/role/[MY_SAGEMAKER_ROLE]/type/api-key* and click *Add*. Click on *Review*, give the policy a name und click on *Create policy*.
+
+.. _hopsworks_aws_policy2.png: ../../_images/aws_policy2.png
+.. figure:: ../../imgs/feature_store/aws_policy2.png
+    :alt: Hopsworks feature store set policy
+    :target: `hopsworks_aws_policy2.png`_
+    :align: center
+    :scale: 30 %
+    :figclass: align-center
+
+**Installing hopsworks-cloud-sdk and connecting to the Feature Store**
+
+To be able to access the Hopsworks Feature Store the hopsworks-cloud-sdk library needs to be installed. One way of achieving this is opening a Python notebook in SageMaker and installing the latest hopsworks-cloud-sdk. Note that the library will not be persistent. For information around how to permanently install a library to Sagemaker see `Install External Libraries and Kernels in Notebook Instances <https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-add-external.html>`_. ::
```

Comment:
@steffengr add `,` after "Feature Store"

is opening -> is by opening

Reply:
done